javascript/javascript-7-oop.md

```diff
@@ -27,6 +27,29 @@
 
 Virtually all companies will use object-oriented programming in their code. For example, when you do a Yelp search for "burgers", each area on the screen that describes each recommended restaurant is an instance of a class. The information is probably abstracted into a class of "Restaurant", of which "Marlowe" and "Super Duper Burgers" are instances of that class.
 
+### Benefits of OOP in JS
+
+- Javascript is mostly used a functional programming language more than an object oriented language like C++, Java. Functional programming in JS can be elegantly used to code and organize the same. The following is a resource to understand the advantages of functional programming in JS.
+
+- [Functional Programming in JS](https://blog.bitsrc.io/functional-programming-in-javascript-how-and-why-94e7a97343b)
+
+- Javascript has a unique OOP style athough its less famous. It can also be used to produce reliable code. Although Javascript hasnt got famous OOP concepts, the frameworks like React, Angular and Vue rely a lot on the same and have got increased popularity amongst developers world wide in many projects.
+
+- [Object oriented programming in React](https://stackoverflow.com/questions/35764800/reactjs-object-oriented)
+
+- Javascript and its frameworks have become must skills for any web developer to master. It's important for students to know for technical interviews as well as to learn other frameworks or languages which have a few ideas based on them. 
```

Comment:
```suggestion
- JavaScript and its frameworks have become must-have skills for any web developer to master. It's important for students to know for technical interviews as well as to learn other frameworks or languages which have a few ideas based on them. 
```
Can you rephrase this sentence?  I don't understand it.  "It's important for students to know for technical interviews as well as to learn other frameworks or languages which have a few ideas based on them. "